include/swift/Serialization/SerializedModuleLoader.h

```diff
@@ -97,7 +97,7 @@ class SerializedModuleLoaderBase : public ModuleLoader {
   openModuleSourceInfoFileIfPresent(AccessPathElem ModuleID,
                                     StringRef ModulePath,
                                     StringRef ModuleSourceInfoFileName,
-                  std::unique_ptr<llvm::MemoryBuffer> *ModuleSourceInfoBuffer);
+      std::unique_ptr<llvm::MemoryBuffer> *ModuleSourceInfoBuffer);
```

Comment:
Ah, sorry, when this left-alignment is necessary it's applied to all arguments for consistency. (We should all have clang-formats lying around…)